src/utils/loose-equal.js

```diff
@@ -29,21 +29,21 @@ function looseEqual(a, b) {
     return true
   }
   const validDatesCount = [isDate(a), isDate(b)].filter(Boolean).length
-  if (validDatesCount > 1) {
+  if (validDatesCount > 0) {
```

Comment:
could this just be `if (validDatesCount) {`

And wondering if we should use a `let validCount` to re-use the same var to prevent a new stack entry for each `const`?

Reply:
1. Don't like that loose checks
2. 👍